### PR TITLE
New package: Baxi.ServiceTool(PCST) version 7.3.15.0

### PR DESCRIPTION
**Pull request by TutkuSilaVuralEyovgeAlten**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137434)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @TutkuSilaVuralEyovgeAlten,

This package seems very similar to another package we have - `Remeha.ServiceTool`. But this is a newer version than its `7.0.53.0`. Here is [the PR](https://github.com/microsoft/winget-pkgs/pull/130389/files) for that version. 

An Internet search says that Remeha is part of Baxi Heating in some way. 

Based on this, should they have the same or similar PackageIdentifiers?

**Comment by stephengillie** [comment]
> Hi @TutkuSilaVuralEyovgeAlten,
> 
> This package seems very similar to another package we have - `Remeha.ServiceTool`. But this is a newer version than its `7.0.53.0`. Here is [the PR](https://github.com/microsoft/winget-pkgs/pull/130389/files) for that version.
> 
> An Internet search says that Remeha is part of Baxi Heating in some way.
> 
> Based on this, should they have the same or similar PackageIdentifiers?

**Comment by TutkuSilaVuralEyovgeAlten** [comment]
> Hi @TutkuSilaVuralEyovgeAlten,
> 
> This package seems very similar to another package we have - `Remeha.ServiceTool`. But this is a newer version than its `7.0.53.0`. Here is [the PR](https://github.com/microsoft/winget-pkgs/pull/130389/files) for that version.
> 
> An Internet search says that Remeha is part of Baxi Heating in some way.
> 
> Based on this, should they have the same or similar PackageIdentifiers?
 
Hello @stephengillie,

You are correct. They are exactly the same packages for different brands with the new version 7.3.15.0. So they need to have different PackageIdentifiers for specific brands. Would it be a problem to create the same packages with different PackageIdentifiers? Because I need to create 2 more packages like this for 2 different brands.

File: manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Baxi.ServiceTool(PCST)
```
Comment (Trenly) [inline manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).installer.yaml:4]:
```suggestion
PackageIdentifier: Baxi.ServiceTool
```

File: manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Baxi.ServiceTool(PCST)
```
Comment (Trenly) [inline manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Baxi.ServiceTool
```

File: manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Baxi.ServiceTool(PCST)
```
Comment (Trenly) [inline manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).yaml:4]:
```suggestion
PackageIdentifier: Baxi.ServiceTool
```

**Review by Trenly — CHANGES_REQUESTED** [review]
After changing the identifier, you will need to rename the folder and files also

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-09T15:12:05.3715889Z ##[error] One or more errors occurred. (Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\b\Baxi\ServiceTool(PCST)\7.3.15.0', expected 'manifests\b\Baxi\ServiceTool\7.3.15.0') Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\b\Baxi\ServiceTool(PCST)\7.3.15.0', expected 'manifests\b\Baxi\ServiceTool\7.3.15.0'

(Automated response - build 723.)

**Comment by stephengillie** [comment]
Oh - the `(PCST)` has to be removed from file name and path, so they match the `PackageIdentifier`.

File: manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Baxi.ServiceTool
```
Comment (stephengillie) [inline manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).installer.yaml:4]: `(PCST)` was removed from the PackageIdentifier. Please update file name and path to match.

File: manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Baxi.ServiceTool
```
Comment (stephengillie) [inline manifests/b/Baxi/ServiceTool(PCST)/7.3.15.0/Baxi.ServiceTool(PCST).locale.en-US.yaml:4]: `(PCST)` was removed from the PackageIdentifier. Please update file name and path to match.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**